### DISCUSSION TITLE
How to disable all existing cookies by default

### DISCUSSION THREAD

**Discussion by amitsarkerr** [orestbida/cookieconsent#627] [General]
How can I disable all the cookies disabled by default?

My idea is, before user's consent, all cookies except necessary categories will be disabled. When user will accept the consent, then the related categories cookies will be activated, otherwise, the cookies will be never active.

**Comment by orestbida** [comment]
If you have configured the scripts/services correctly, there should be no cookies set, unless the user accepts that service/category.
